imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml

```diff
@@ -2315,103 +2424,120 @@ esaa_dac:
     <<: *hskp_default
     LABLAXIS: OPTICS_HV_DAC_ESA_A
     FIELDNAM: Optics HV - ESA A DAC
+    FILLVAL: *uint16_fillval
     CATDESC: Optics HV - ESA A DAC
 
 esab_dac:
     <<: *hskp_default
     LABLAXIS: OPTICS_HV_DAC_ESA_B
     FIELDNAM: Optics HV -  ESA B DAC
+    FILLVAL: *uint16_fillval
     CATDESC: Optics HV -  ESA B DAC
 
 iobulk_dac:
     <<: *hskp_default
     LABLAXIS: OPTICS_HV_DAC_IONBULK
     FIELDNAM: Optics HV - Ion Bulk DAC
+    FILLVAL: *uint16_fillval
     CATDESC: Optics HV - Ion Bulk DAC
 
 ssdo_dac:
     <<: *hskp_default
     LABLAXIS: SENSOR_HV_DAC_SSDO
     FIELDNAM: Sensor HV - SSDO Enable
+    FILLVAL: *uint16_fillval
     CATDESC: Sensor HV - SSDO Enable
 
 ssdb_dac:
     <<: *hskp_default
     LABLAXIS: SENSOR_HV_DAC_SSDB
     FIELDNAM: Sensor HV - SSD Bias Enable
+    FILLVAL: *uint16_fillval
     CATDESC: Sensor HV - SSD Bias Enable
 
 apdb_dac:
     <<: *hskp_default
     LABLAXIS: SENSOR_HV_DAC_APDB
     FIELDNAM: Sensor HV - ADP Bias Enable
+    FILLVAL: *uint16_fillval
     CATDESC: Sensor HV - ADP Bias Enable
 
 apdb2_dac:
     <<: *hskp_default
     LABLAXIS: SENSOR_HV_DAC_APDB2
     FIELDNAM: Sensor HV - ADP Bias 2 Enable
+    FILLVAL: *uint16_fillval
     CATDESC: Sensor HV - ADP Bias 2 Enable
 
 strmcp_dac:
     <<: *hskp_default
     LABLAXIS: SENSOR_HV_DAC_START_MCP
     FIELDNAM: Sensor HV - Start MCP DAC
+    FILLVAL: *uint16_fillval
     CATDESC: Sensor HV - Start MCP DAC
 
 stpmcp_dac:
     <<: *hskp_default
     LABLAXIS: SENSOR_HV_DAC_STOP_MCP
     FIELDNAM: Sensor HV - Stop MCP DAC
+    FILLVAL: *uint16_fillval
     CATDESC: Sensor HV - Stop MCP DAC
 
 stpog_dac:
     <<: *hskp_default
     LABLAXIS: SENSOR_HV_DAC_STOP_OPTICS_GRID
     FIELDNAM: Sensor HV - Stop Optics Grid DAC
+    FILLVAL: *uint16_fillval
     CATDESC: Sensor HV - Stop Optics Grid DAC
 
 sbulk_vmon:
     <<: *hskp_default
     LABLAXIS: SBULK_VMON
-    FIELDNAM: HVPS – V1 -- Sensor Bulk Voltage Monitor
```

Comment:
A lot of these changes here are because the dash (`-`) that I had was some sort of copy/paste issue that was not of supported unicode by SDPF.